### ISSUE TITLE
Add pitch deck and supporting documents in Wiki

### ISSUE THREAD

**Issue by yashbhutoria** [Azure/azure-service-bus#345]
Supporting documents, pitch decks, deep dive presentations and PDFs should be added in the Wiki.

**Comment by SeanFeldman** [comment]
I will never say no to additional documentation. 👍 on that side.
When it comes to where to have the documents, wiki is sub-optimal. The documentation should be maintained and be part of the official documentation vast majority of customers/potential users are looking at - [Microsoft docs site](https://docs.microsoft.com/azure/service-bus-messaging/).

You've sort of threw in a pile various things. That's not productive. And here's why:
- Support documents - what does that mean? Features? How does the broker work? How to send/receive?
- Pitch decks - who's the target? Developers? Tech leads? CTOs?
- Deep dive presentations - again, the devil is in details. Community presentations? Official presentations? Who'll maintain and update those?
- PDFs - what's the value of static PDFs over living and breathing document that can be easily updated and kept up to date on the site?

To sum it up, I fully support your ask for better documentation. If you're serious about it, don't drop a single liner "do X", elaborate and provide specifics. It will help your cause, as well as the product group with understanding what needs to be improved. Cheers.

**Comment by axisc** [comment]
@yashbhutoria  - thanks for raising this issue. Happy to take your feedback on improving the documentation.

In terms of product pitch and supporting documentation, our current docs, blogs and channel 9 videos should be sufficient to help drive customer conversations forward. 

Please let us know if there was anything specific you were looking for but didn't see it.

**Comment by yashbhutoria** [comment]
@SeanFeldman @axisc I was suggesting something similar to azure functions repo
https://github.com/Azure/Azure-Functions/wiki/Samples-and-content

**Comment by SeanFeldman** [comment]
Half of that stuff (if no more) is out of date. Sample from 2017.
All the wiki should be part of the official documentation if not already.
I've personally had a few cases when a wiki was out of date and needed to chase people down to get the right info. There are better ways to document and capture things. Wiki is not one of those IMO.
